### PR TITLE
ci: package and release helm charts

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -100,6 +100,19 @@ jobs:
             options+=('--tags' '${{ github.event.inputs.tag }}')
           fi
           gren release --draft "${options[@]}" --username=${{ github.repository_owner }} --repo=${{ github.event.repository.name }} --token=${{ secrets.GITHUB_TOKEN }}
+      - uses: azure/setup-helm@v1
+      - run: helm package helm-charts/move2kube
```
Comment (ashokponkumar) [inline .github/workflows/release.yml:104]: We need to update the version number in values.yaml before packaging the chart.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -100,6 +100,19 @@ jobs:
...
+      - run: mkdir temp/
+      - run: mv *.tgz temp/
+      - run: curl -L -o oldindex.yaml https://konveyor.io/move2kube/index.yaml
+      - run: helm repo index --merge oldindex.yaml --url https://github.com/konveyor/move2kube-operator/releases/download/${{ github.event.inputs.tag }} temp/
+      - uses: HarikrishnanBalagopal/upload-release-action@v3
```
Comment (ashokponkumar) [inline .github/workflows/release.yml:109]: Can we move this repository to konveyor too? We are using in multiple places and its a useful reusable action for konveyor.

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Also, while branching we need to change the version number in the values.yaml in the branch to use the right image name.

File: Makefile [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+	IMAGE_TAG := ${GIT_BRANCH}
+endif
+ifeq ($(IMAGE_TAG),)
+	IMAGE_TAG := latest
+endif
+ifeq ($(IMAGE_TAG),main)
+	IMAGE_TAG := latest
+endif
+
+RELEASE_NAME ?= myproject
```
Comment (ashokponkumar) [inline Makefile:15]: Can we set the release name as move2kube by default

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (HarikrishnanBalagopal) [reply Makefile]: done

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**